third_party/vmServiceDrivers/org/dartlang/vm/service/element/Field.java

```diff
@@ -67,14 +68,20 @@ public ObjRef getOwner() {
   }
 
   /**
-   * The value of this field, if the field is static.
+   * The value of this field, if the field is static. If uninitialized, this will take the value of
+   * an uninitialized Sentinel.
+   *
+   * @return one of <code>InstanceRef</code> or <code>Sentinel</code>
```

Comment:
This claims Sentinel can be returned but the code actually returns null in that case.

Reply:
Hmm, yes, from reading the code that looks right - the comment is out of sync w/ actual implementation.

I think this is an issue w/ the upstream code generator - the majority of this Java library is generated; https://github.com/dart-lang/sdk/blob/master/pkg/vm_service/tool/java/generate_java.dart.